README.md

```diff
@@ -43,7 +43,15 @@ If you get stuck while setting up your own environment, you can ask questions in
 
 ### Push Notifications
 
-If you want to work on push notifications or use a development build with push notifications please go the server side code [HERE](https://github.com/home-assistant/mobile-apps-fcm-push) and deploy it to your firebase project.  Once you have your androidV1 URL to the deployed service, exchange that for your local builds [PUSH_URL](https://github.com/home-assistant/android/blob/master/common/src/main/java/io/homeassistant/companion/android/common/data/integration/impl/IntegrationRepositoryImpl.kt#L43).
+If you want to work on push notifications or use a development build with push notifications, please go the server side code [HERE](https://github.com/home-assistant/mobile-apps-fcm-push) and deploy it to your firebase project. Once you have your androidV1 URL to the deployed service, set it in to your `${GRADLE_USER_HOME}/gradle.properties` file, e.g.:
+```properties
+homeAssistantAndroidPushUrl=https://mydomain.cloudfunctions.net/androidV1
```

Comment:
I prefixed the properties with `homeAssistantAndroid` to make it clear which project uses those when overridden in the user's `~/.gradle/gradle.properties`.